### PR TITLE
Pre merge

### PR DESCRIPTION
**Pull request by CoquilledeLune**
Thank you Thomas ! :)

### CODE REVIEW

File: src/View/Shared/partnersForUsers.html.twig [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+              <p class="card-text">{{partner.address}}</p>
+              <p class="card-text">{{partner.email}}</p>
+              <p class="card-text">{{partner.phone}}</p>
+              <button class="col-4 p-1">
+              <a class="fs-5" href="/onePartner?id={{partner.id}}">Voir mes produits</a>
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
```
Comment (busy-gnl) [inline src/View/Shared/partnersForUsers.html.twig:28]: you could put that card in a component

**Review by aldaitzwild — CHANGES_REQUESTED** [review]
Quality checks failed

**Comment by CoquilledeLune** [comment]
Should be ok now

File: src/Model/PartnerManager.php [hunk 2]
```diff
@@ -39,4 +39,14 @@ public function update(array $partner): bool
 
         return $statement->execute();
     }
+
+    public function showPartnerWine(int $id): array
```
Comment (aldaitzwild) [inline src/Model/PartnerManager.php:43]: This is a bad name for this function, it should somtehing like "getWinesByPartner"

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

**Review by aldaitzwild — APPROVED** [review]

**Pull request merged**